### PR TITLE
Create a new publishing task in gradle

### LINKED ISSUE #195: Use maven publish plugin for release
https://docs.gradle.org/current/userguide/publishing_maven.html

### PR DESCRIPTION
**Pull request by jingtang10**
This will be used to generate the artifacts to be published on maven.

Closes #195

### CODE REVIEW

File: datacapture/build.gradle [hunk 1]
```diff
@@ -1,6 +1,31 @@
...
+
+afterEvaluate {
+    publishing {
+        publications {
+            release(MavenPublication) {
+                from components.release
+
+                groupId = 'com.google.android.fhir'
+                artifactId = 'data-capture'
+                version = '0.1.0-alpha1'
```
Comment (yigit) [inline datacapture/build.gradle:15]: alpha01

Comment (jingtang10) [reply datacapture/build.gradle]: done

**Comment by jingtang10** [comment]
@yigit I updated this PR to include source code in the release. Also marked a bunch of classes as internal. PTAL. Thanks!

**Review by yigit — APPROVED** [review]

**Review by joiskash — APPROVED** [review]

**Comment by jingtang10** [comment]
Thanks both!

**Pull request merged**